platform-sdk/swirlds-jasperdb/src/main/java/com/swirlds/merkledb/files/hashmap/Bucket.java

```diff
@@ -251,7 +232,8 @@ public void putValue(final int keyHashCode, final K key, final long value) {
                         // move all entries after this one up
                         final int offsetOfNextEntry = result.entryOffset + entrySize;
                         final int sizeOfEntriesToMove = currentSize - offsetOfNextEntry;
-                        //  FUTURE WORK For Java 17 do this https://github.com/swirlds/swirlds-platform/issues/4090
+                        //  FUTURE WORK For Java 17 do this
+                        // https://github.com/swirlds/swirlds-platform/issues/4090
```

Comment:
Definitely shouldn't keep a reference to an issue in a different repo in the code here.

Reply:
There are many references like this all over the code. As far as I understand, most issues will be migrated from swirlds/swirlds-platform to hashgraph/hedera-services shortly. After that, we can (should) change all such links in the sources.